### PR TITLE
Doc strings reformatting

### LINKED ISSUE #142: checking/updating doc strings
Add docstrings to utility scripts and get them their set of pages on readthedocs

### PR DESCRIPTION
**Pull request by mschwamb**
Reformat doc strings so that they match the required format for the LINCC Framesworks incubator template setup for our documentation sphinx setup (which follows the numpy standard https://numpydoc.readthedocs.io/en/latest/format.html)

Fixes #142

### CODE REVIEW

**Comment by mschwamb** [comment]
All docstrings are standardized as close as I can get them and follow the NumPy format so they now render in read the docs.

File: src/sorcha/ephemeris/simulation_driver.py [hunk 1]
```diff
@@ -74,14 +60,30 @@ def create_ephemeris(orbits_df, pointings_df, args, configs):
...
+
+    Notes
+    -------
+    This works by calculating and regularly updating the sky-plane
+    locations (unit vectors) of all the objects in the collection
+    of orbits.  The HEALPix index for each of the locations is calculated.
+    A dictionary with pixel indices as keys and lists of ObjIDs for
+    those objects in each HEALPix tile as values.  One of these
+    calculations is called a 'picket', as one element of a long picket
+    fence.  At present,
```
Comment (astronomerritt) [inline src/sorcha/ephemeris/simulation_driver.py:80]: There's an unfinished sentence here.

Comment (mschwamb) [reply src/sorcha/ephemeris/simulation_driver.py:80]: It was in there to begin with. So I think let's leave it and let it get addressed in #763

**Review by astronomerritt — APPROVED** [review]
Amazing. Must have been a lengthy and tedious job!

**Pull request merged**